modules/cco-ccoctl-configuring.adoc

```diff
@@ -111,6 +111,28 @@ ifdef::aws-sts[]
 * `s3:PutObjectAcl`
 * `s3:PutObjectTagging`
 
+|====
++
+If you plan to store the OIDC configuration in a private S3 bucket that is accessed by the IAM identity provider through a public CloudFront distribution URL, the AWS account requires the following additional permissions:
```

Comment:
> the AWS account requires

What about "the AWS User/Role running ccoctl requires[...]" ?

Reply:
cut out some of the repeat verbiage for sentence length, but will add it back in for clarity :)